datareservoirio/_logging.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import os
+from functools import wraps
+
+from opencensus.ext.azure.log_exporter import AzureLogHandler
+
+import datareservoirio as drio
+
+from .globalsettings import environment
+
+exceptions_logger = logging.getLogger(__name__ + "_exception_logger")
+exceptions_logger.setLevel(logging.INFO)
+
+if os.getenv("APPLICATION_INSIGHTS_LOGGER") is not None:
+    if os.environ["APPLICATION_INSIGHTS_LOGGER"] == "true":
```

Comment:
It could be enough to just check that it is not None, that is that some variable is set. I have seen that used sometimes, like a flag without a value.

Reply:
```DRIO_PYTHON_APPLICATION_INSIGHTS_LOGGER```

Reply:
DRIO_PYTHON_APPINSIGHTS

Reply:
If we are to check for true, we must do a case-insensitive check 
True
true
TRUE
and 
1